### PR TITLE
Support headless config

### PR DESCRIPTION
**Pull request by allanwebix**
To support non headless mode in the `ENV` config file

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -215,7 +215,7 @@ const sleepingTime = sleepingTimeInMinutes * 60000;
         try {
             console.log('START ', process.env.ACCOUNT, new Date().toLocaleString())
             const browser = await puppeteer.launch({
-                headless: true,
+                headless: process.env.HEADLESS || true,
```
Comment (alfficcadenti) [inline index.js:218]: Thanks for contributing. 
I like the idea, but did you test it?
I think that when the variable is set to false, this condition will become true.
Also, I think the variables from the process.env are all read as string so in this case it won't be a boolean as expected by puppeteer.

I suggest creating another variable for the logic. Check how I did here for the quest:
https://github.com/alfficcadenti/splinterlands-bot/blob/bdc88d188edcff654fe1ba339fbc34f52a4b4c79/possibleTeams.js#L224

File: README.md [hunk 2]
```diff
@@ -40,6 +41,8 @@ Example:
 
 - `CLAIM_SEASON_REWARD=true`
 
+- `HEADLESS=false`
```
Comment (alfficcadenti) [inline README.md:44]: thanks for updating the readme. Can you please add it to the example env file?

File: index.js (hunk 1 above)
Comment (allanwebix) [reply index.js]: Yup, I tested it, and I updated it

Comment (alfficcadenti) [reply index.js]: 👍 resolve the conflicts and I can merge it

**Review by alfficcadenti — APPROVED** [review]
:+1:

**Pull request merged**